tests/gem5/traffic_gen/configs/simple_traffic_run.py

```diff
@@ -53,7 +53,7 @@ def generator_factory(
 
         return LinearGenerator(
             duration="250us",
-            rate="40GB/s",
+            rate="40GiB/s",
```

Comment:
Ditto here for bandwidth

Reply:
I think this should also be okay to change, since LinearGeneratorCore uses `toMemoryBandwidth()`, which converts base 10 to base 2. RandomGeneratorCore also uses `toMemoryBandwidth()`.